content/en/serverless/guide/gcr_serverless_init.md

```diff
@@ -0,0 +1,194 @@
+
+First, manually install your tracer:
+- [Node.JS][14]
+- [Python][13]
+- [Java][15]
+- [Go][12]
+- [.NET][18]
+- [Ruby][16]
+- [PHP][17]
+
+Then, build your application by running the following command:
+
+```shell
+pack build --builder=gcr.io/buildpacks/builder \
+--buildpack from=builder \
+--buildpack datadog/serverless-buildpack:latest \
+gcr.io/YOUR_PROJECT/YOUR_APP_NAME
+```
+
+**Note**: Buildpack instrumentation is not compatible with Alpine images
+
+## Configure your application
+
+Once the container is built and pushed to your registry, the last step is to set the required environment variables for the Datadog Agent:
+- `DD_API_KEY`: Datadog API key, used to send data to your Datadog account. It should be configured as a [Google Cloud Secret][11] for privacy and safety issue.
+- `DD_SITE`: Datadog endpoint and website. Select your site on the right side of this page. Your site is: {{< region-param key="dd_site" code="true" >}}.
+- `DD_TRACE_ENABLED`: set to `true` to enable tracing
+- `DD_TRACE_PROPAGATION_STYLE`: Set this to `datadog` to use context propagation and log trace correlation.
+
+For more environment variables and their function, see [Additional Configurations](#additional-configurations).
+
+The following command deploys the service and allows any external connection to reach it. Set `DD_API_KEY` as an environment variable, and set your service listening to port 8080.
+
+```
+shell
+gcloud run deploy APP_NAME --image=gcr.io/YOUR_PROJECT/APP_NAME \
+  --port=8080 \
+  --update-env-vars=DD_API_KEY=$DD_API_KEY \
+  --update-env-vars=DD_TRACE_ENABLED=true \
+  --update-env-vars=DD_SITE='datadoghq.com' \
+  --update-env-vars=DD_TRACE_PROPAGATION_STYLE='datadog' \
+```
+
+## Results
+
+Once the deployment is completed, your metrics and traces are sent to Datadog. In Datadog, navigate to **Infrastructure->Serverless** to see your serverless metrics and traces.
+
+## Additional configurations
+
+- **Advanced Tracing:** The Datadog Agent already provides some basic tracing for popular frameworks. Follow the [advanced tracing guide][2] for more information.
+
+- **Logs:** If you use the [Google Cloud integration][1], your logs are already being collected. Alternatively, you can set the `DD_LOGS_ENABLED` environment variable to `true` to capture application logs through the serverless instrumentation directly.
+
+- **Custom Metrics:** You can submit custom metrics using a [DogStatsd client][4]. For monitoring Cloud Run and other serverless applications, use [distribution][9] metrics. Distributions provide `avg`, `sum`, `max`, `min`, and `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also manage tags. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][11]. To monitor a distribution for a count metric type, use `sum` for both the time and space aggregations.
+
+### Environment Variables
```

Comment:
Can we add this table to the sidecar cloud run page? This is only in the old guide, and there isn't any mention of the [additional tags](https://docs-staging.datadoghq.com/cswatt/gcr_sidecar/getting_started/tagging/unified_service_tagging/) like `DD_VERSION`